zipline/pipeline/expression.py

```diff
@@ -328,7 +330,12 @@ def __repr__(self):
 
     def short_repr(self):
         """Short repr to use when rendering Pipeline graphs."""
-        return "Expression: {expr}".format(
+        final = self._expr
+        num_list = re.findall(r"[-+]?\d*\.\d+", self._expr)
+        for i in num_list:
+            final = final.replace(i, '%.2E' % Decimal(i))
```

Comment:
How does the loop compare to `re.sub`? I could see it being potentially more efficient, but I don't know if it matters here.

Reply:
Oh I'll use that instead - str.replace() is an old habit I have from Java